nuxeo-features/nuxeo-platform-oauth/src/main/java/org/nuxeo/ecm/platform/oauth2/request/TokenRequest.java

```diff
@@ -44,13 +45,16 @@
 
     protected String codeVerifier;
 
+    protected String assertion;
```

Comment:
Missing `@since`.

Reply:
For `protected` (not part of the API) it's not strictly needed

Reply:
I always assume that `public` and `protected` are part of our API as we are building a platform to be extended 😃